packages/web-pkg/src/composables/sse/useServerSentEvents.ts

```diff
@@ -0,0 +1,35 @@
+import { EventSourceMessage, fetchEventSource } from '@microsoft/fetch-event-source'
+import { unref } from 'vue'
+import { v4 as uuidV4 } from 'uuid'
+import { useGettext } from 'vue3-gettext'
+import { configurationManager, useAccessToken, useStore } from 'web-pkg/src'
+export * from './useServerSentEvents'
+
+export interface ServerSentEventsOptions {
+  url: string
+  onOpen?: (response: Response) => void
+  onMessage?: (msg: EventSourceMessage) => void
+}
+
+export const useServerSentEvents = (options: ServerSentEventsOptions) => {
+  const store = useStore()
+  const language = useGettext()
+  const accessToken = useAccessToken({ store })
+  const setupServerSentEvents = async () => {
+    await fetchEventSource(`${configurationManager.serverUrl}${options.url}`, {
```

Comment:
`path.join` doesn't work here the output is something like "https:/host.docker..." which is wrong